```
/Volumes/Data/worker/tvOS-26-Simulator-Build-EWS/build/WebKitBuild/Release-appletvsimulator/DerivedSources/WebCore/JSDOMWindow.cpp:257:10: error: 'JSFontFaceSetLoadEvent.h' file not found (in target 'WebCore' from project 'WebCore')
    /Volumes/Data/worker/tvOS-26-Simulator-Build-EWS/build/WebKitBuild/Release-appletvsimulator/DerivedSources/WebCore/unified-sources/UnifiedSource7.cpp:57:10: note: in file included from /Volumes/Data/worker/tvOS-26-Simulator-Build-EWS/build/WebKitBuild/Release-appletvsimulator/DerivedSources/WebCore/unified-sources/UnifiedSource7.cpp:57:
```
This build error at "tv-sim" is not occurred by my change.
@whsieh , would you merge this PR ...? I don't have the permission to merge.

--- a/Source/WebKit/UIProcess/TextExtractionAssertionScope.cpp
+++ b/Source/WebKit/UIProcess/TextExtractionAssertionScope.cpp
@@ -26,6 +26,8 @@
 #include "config.h"
 #include "TextExtractionAssertionScope.h"
 
+#include <wtf/TZoneMallocInlines.h>
+
 namespace WebKit {
 
 WTF_MAKE_TZONE_ALLOCATED_IMPL(TextExtractionAssertionScope);

